CMakeLists.txt

```diff
@@ -185,8 +188,7 @@ maketest(message-alignment)
 maketest(test-arg-iter)
 if(LIBLO_FOUND)
     maketest(liblo)
-    target_include_directories(liblo PRIVATE ${LIBLO_INCLUDE_DIRS})
```

Comment:
Why removing the include directory?

Reply:
The include dirs are being found by liblo in their master branch's integration. Also, the LIBLO_INCLUDE_DIRS variable is not working anymore because we are not having the custom findliblo.cmake

Reply:
So the CMake's FindLiblo does not set `LIBLO_INCLUDE_DIRS`?